addons/procurement/product_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just to follow the guidelines: rename to product_product_views.xml. Or product_template_views.xml . I don't know which one is the "best of the top of the best" model between product_product and product_template. Somehow, you will have to choose ... good luck.